src/plugins/cookies/cookie.js

```diff
@@ -0,0 +1,38 @@
+import URI from 'URIjs';
+
+export default class ChromeCookie {
+
+    constructor (details) {
```

Comment:
Why there is a space between function name and a left brace in class methods but not in stand alone functions?